app/src/main/res/layout/search_coin_preview.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <ImageView
+        android:id="@+id/ivSearchIcon"
```

Comment:
I would probably drop the `iv` and just name it `searchIcon`? Icon implies it is some kind of view that display s images anyway and with databinding/ viewbinding the type if the view is inferred anyway :D